--- linux/PKGBUILD	2012-12-13 21:04:21.000000000 -0800
+++ PKGBUILD	2012-12-13 22:28:11.298313524 -0800
@@ -2,7 +2,7 @@
 # Maintainer: Tobias Powalowski <tpowa@archlinux.org>
 # Maintainer: Thomas Baechler <thomas@archlinux.org>
 
-pkgbase=linux               # Build stock -ARCH kernel
+pkgbase=linux-ib            # Build stock -ARCH kernel
 #pkgbase=linux-custom       # Build kernel with a different name
 _srcname=linux-3.6
 pkgver=3.6.10
@@ -10,7 +10,6 @@
 arch=('i686' 'x86_64')
 url="http://www.kernel.org/"
 license=('GPL2')
-makedepends=('xmlto' 'docbook-xsl')
 options=('!strip')
 source=("http://www.kernel.org/pub/linux/kernel/v3.x/${_srcname}.tar.xz"
         "http://www.kernel.org/pub/linux/kernel/v3.x/patch-${pkgver}.xz"
@@ -23,6 +22,7 @@
         'module-init-wait-3.6.patch'
         'irq_cfg_pointer-3.6.6.patch'
         'fat-3.6.x.patch')
+# cksum start
 md5sums=('1a1760420eac802c541a20ab51a093d1'
          '406a52f90a2ddc78a3ecdf4fe46e7cf7'
          '65f7ff39775f20f65014383564d3cb65'
@@ -33,6 +33,7 @@
          '8a71abc4224f575008f974a099b5cf6f'
          '4909a0271af4e5f373136b382826717f'
          '88d501404f172dac6fcb248978251560')
+# cksum end
 
 _kernelname=${pkgbase#linux}
 
@@ -319,7 +320,7 @@
   rm -f "${pkgdir}/usr/src/linux-${_kernver}/Documentation/DocBook/Makefile"
 }
 
-pkgname=("${pkgbase}" "${pkgbase}-headers" "${pkgbase}-docs")
+pkgname=("${pkgbase}" "${pkgbase}-headers")
 for _p in ${pkgname[@]}; do
   eval "package_${_p}() {
     _package${_p#${pkgbase}}
